tools/codegen/base/cargo-lambda.json

```diff
@@ -0,0 +1,16 @@
+{
+  "repository": "https://github.com/cargo-lambda/cargo-lambda",
+  "tag_prefix": "v",
+  "rust_crate": "${package}",
+  "asset_name": "${package}-v${version}.${rust_target}.tar.gz",
+  "version_range": ">= 1.3.0",
+  "platform": {
+    "x86_64_linux_musl": {},
+    "x86_64_macos": {},
+    "x86_64_windows": {
+      "asset_name": "${package}-v${version}.windows-x64.zip",
+      "bin": "${package}${exe}"
+    },
+    "aarch64_macos": {}
```

Comment:
It seems they distribute the aarch64_musl binary:
https://github.com/cargo-lambda/cargo-lambda/releases/tag/v1.3.0
> cargo-lambda-v1.3.0.aarch64-unknown-linux-musl.tar.gz